.tekton/serverless-bundle-pull-request.yaml

```diff
@@ -0,0 +1,441 @@
+apiVersion: tekton.dev/v1
```

Comment:
😮 that is a lot of yaml for a few builds with a lot of repetition. Any way to re-use stuff across pipelineruns?

Reply:
I was looking for something similar, but didn't find anything initially. I only saw, that this works for tasks (as done in [PAC repo](https://github.com/openshift-pipelines/pipelines-as-code/tree/main/.tekton) for example) - but not for some pipeline run templates (maybe there is something Tekton "native" :shrug: ).